certbot_external_auth/plugin.py

```diff
@@ -724,6 +727,26 @@ def _call_handler(self, command, *args, **kwargs):
     # Helper methods & UI
     #
 
+    def _json_sanitize_dict(self, dictionary):
+        for key, val in list(dictionary.items()):
+            # Not highly effecient, would be neater to clean up FIELD_TOKEN.
+            # But if any of the others turn to bytes in the future, this will solve it:
+            if type(key) == bytes:
```

Comment:
I thought so to but wasn't 100% sure. But since we define the keys ourselves that in of itself is a grantee they should never be bytes. We can remove it!